### DISCUSSION TITLE
[Unity] 예진의 Layout group & Layout Element

### DISCUSSION THREAD

**Discussion by pnudpwls1103** [BDD-CLUB/challenge-every-week-posting#128] [1. 개발 관련 글 모음]
이번주는 몸이 안 좋아서 퇴근하고 집에서 계속 누워있어서 글을 급하게 썼네요 ㅠㅠ..
원래 글에 더 쓰고 싶은 내용들이 있는데 자료조사랑 공부가 덜 돼서 나중으로 미루다보니 글 퀄리티도 떨어지고 나중에 쓸 글이 불어나는거 같아요ㅠㅠ 
그리고 자료조사 후에 원래 글에 내용을 보충할지 게시글을 따로 올리지도 좀 고민되네요.. 

아직 낯선 환경에 적응이 안 돼서 그런지 주말 루틴이 안 잡혀서 할 일만 쌓이고 있습니다 ㅎㅎ..

[블로그 링크](https://donkeysdevelpment.tistory.com/130)

**Comment by llddang** [comment]
오옹 unity 아직 안 접해봤지만 그림 있어서 알아보기 쉽당!
그리고 뭔가 3학년때 배운 앱스튜디오 생각낰ㅋㅋㅋㅋㅋㅋ

우잉... 홧팅홧팅!!

**Reply by pnudpwls1103** [reply]
알아보기 쉬워서 다행이땅!!!
